### DISCUSSION TITLE
What help with any code

### DISCUSSION THREAD

**Discussion by AkshatGoyal1018** [Newbyes/Hackerrank#5] [Q&A] [answered]
In Akshat/java/week2 hashsetpro.java cannot pass 2 test cases because is was taking more time. Can anyone help to optimize the code.

**Comment by AkshatGoyal1018** [comment, marked as answer]
Got it. I used the simple normal way to do it. But the question also said to use hash set. Normal way was right but lengthy. So, hashsetpro2 is with hash set with all the test cases passed. ;)

**Reply by PHAGUN-JAIN** [reply]
XD
